src/components/AppBar/AppBar.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+	import type { Snippet } from "svelte";
+
+	interface AppBarProps {
+		/* Styling */
+		rootBackground?: string;
+		rootBorder?: string;
+		rootPadding?: string;
+		rootShadow?: string;
+		rootSpacing?: string;
+		rootGridColumns?: string;
+		rootGap?: string;
+		regionRowMain?: string;
+		regionRowHeadline?: string;
+		leadSnippetClasses?: string;
+		defaultSnippetClasses?: string;
+		trailSnippetClasses?: string;
+
+		/* Snippets */
+		regionRowHeadlineSnippet?: Snippet;
+		leadSnippet?: Snippet;
+		children?: Snippet;
+		trailSnippet?: Snippet;
+
+		/* Accessibility */
+		label?: string;
+		labelledby?: string;
+
+		/* Rest */
+		class?: string;
+	}
+
+	let {
+		rootBackground = "bg-surface-100-800-token",
+		rootBorder = '',
+		rootPadding = 'p-4',
+		rootShadow = '',
+		rootSpacing = "space-y-4",
+		rootGridColumns = "grid-cols-[auto_1fr_auto]",
+		rootGap = "gap-4",
+		regionRowMain = '',
+		regionRowHeadline= '',
+		leadSnippetClasses= '',
+		defaultSnippetClasses= '',
+		trailSnippetClasses= '',
+		regionRowHeadlineSnippet,
+		leadSnippet,
+		children: defaultSnippet,
+		trailSnippet,
+		label = '',
+		labelledby = '',
+		/* Unsure what to alias this as "class" is reserved in js, alternatives: className, restClass, restClassName, clazz */
+		class: className = ''
+	} = $props<AppBarProps>();
+
+	const joined = (...items: string[]) => items.join(" ");
```

Comment:
The default styles per-element should now live in the `fooBase` style prop provided in the props list above. No more `cBase` classes. See how we handled this for the `footerBase` in the Card component:
https://github.com/skeletonlabs/skeleton-next/blob/main/src/components/Card/Card.svelte#L41

Reply:
Resolved